lib/Routing/Element/Router.php

```diff
@@ -56,8 +56,11 @@ public function __construct(RequestContext $context, RequestHelper $requestHelpe
 
     /**
      * {@inheritdoc}
+     *
+     * @return array
      */
-    public function setContext(RequestContext $context)
+    #[\ReturnTypeWillChange]
+    public function setContext(RequestContext $context)//: array
```

Comment:
```suggestion
    public function setContext(RequestContext $context)//: void
```

Reply:
Ohh...thank you!! I think it was changed by mistake as there was no deprecation related to this. I'll revert it 😉